spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe EmsCloudHelper do
+  before do
+    controller.send(:extend, EmsCloudHelper)
+    self.class.send(:include, EmsCloudHelper)
+  end
+
+  context "textual_instances" do
+    def role_allows(_)
+      true
+    end
```

Comment:
@matthewd Thoughts?  I see it's calling `extend` on an instance, which *should* create them like instance methods, right?  So not sure why the stub wouldn't work.